api/turing/models/ensembling_job.go

```diff
@@ -76,11 +76,11 @@ func (r *InfraConfig) Scan(value interface{}) error {
 
 // BatchEnsemblingJobResources contains the resources delared to run the ensembling job.
 type BatchEnsemblingJobResources struct {
-	DriverCPURequest      string `json:"driver_cpu_request,omitempty"`
```

Comment:
Why has the `omitempty` been removed?

Reply:
I don't think there is a reason to do that anymore since the values are going to be populated